assets/src/edit-story/output/utils/styles.js

```diff
@@ -22,11 +22,16 @@ import { FULLBLEED_RATIO, PAGE_RATIO } from '../../constants';
 function CustomStyles() {
   const safeToFullRatio = PAGE_RATIO / FULLBLEED_RATIO;
   const fullToSafeRatio = 1 / safeToFullRatio;
+  const pageBackgroundColor = '#1b1d1c'; /* theme.colors.bg.workspace */
```

Comment:
You can import the default export from `../../theme` and load it from there - then it would just be:

```js
import theme from '../../theme';
...
  const pageBackgroundColor = theme.colors.bg.workspace;
```

Reply:
Thanks! Sorry for the delay, I'll fix this today.